ios/RNInstabug/InstabugReactBridge.m

```diff
@@ -24,10 +24,7 @@ + (void)setWillSendReportHandler_private:(void(^)(IBGReport *report, void(^repor
 @implementation InstabugReactBridge
 
 - (NSArray<NSString *> *)supportedEvents {
-    return @[
-             @"IBGpreSendingHandler",
-             @"IBGSetNetworkDataObfuscationHandler",
-             ];
+    return @[@"IBGpreSendingHandler"];
```

Comment:
**suggestion:** I think the `supportedEvents` is redundant and can be replaced with a more useful enum (same as the android suggestion above). What do you think?